pkg/operator/ceph/cluster/nodedaemon/add.go

```diff
@@ -42,7 +42,7 @@ import (
 const (
 	controllerName = "ceph-nodedaemon-controller"
 	// AppName is the value to the "app" label for the ceph-crash pods
-	crashCollectorAppName = "rook-ceph-crashcollector"
+	CrashCollectorAppName = "rook-ceph-crashcollector"
```

Comment:
Why is this renamed? Looks like it's not used in another package.

Reply:
I see, so you could revert it, right? Sounds ok to use the string directly if you're getting a build error.